### PR TITLE
MNT Use shared travis config, use sminnee/phpunit

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,48 +1,4 @@
...
-  # Init PHP
-  - phpenv rehash
-  - phpenv config-rm xdebug.ini
-  - echo 'memory_limit = 2048M' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-
-  # Install composer dependencies
-  - composer validate
-  - composer require --no-update silverstripe/recipe-cms:$RECIPE_VERSION
-  # This module has integration testing requirements
-  - composer require --no-update silverstripe/subsites:^2 silverstripe/taxonomy:^2 silverstripe/contentreview:^4
```
Comment (dnsl48) [inline .travis.yml:36]: Weird... how does it work without these dependencies now?

**Review by dnsl48 — CHANGES_REQUESTED** [review]
Need to investigate what happens about the extra requirements?
```
composer require --no-update silverstripe/subsites:^2 silverstripe/taxonomy:^2 silverstripe/contentreview:^4
```

File: .travis.yml (hunk 1 above)
Comment (emteknetnz) [reply .travis.yml:36]: The following has been added:
```
env:
  global:
    - REQUIRE_EXTRA="silverstripe/subsites:^2 silverstripe/taxonomy:^2 silverstripe/contentreview:^4"
```

**Comment by emteknetnz** [comment]
This one is still failing because of yml quoting in subsites.  It's requiring it as subsites:^2 and the subsites with yml fix hasn't been released yet

I think we should just merge this one as is

**Review by dnsl48 — APPROVED** [review]

**Pull request merged**